features/prefers-reduced-data.yml

```
@@ -0,0 +1,6 @@
+name: prefers-reduced-data media query
+description: The `prefers-reduced-data` CSS media query detects whether the user has a preference for using less internet traffic. For example, you can use this media query to avoid loading large font files and use a system font instead.
```

me dodging having to make a decision about Internet or internet:

```suggestion
description: The `prefers-reduced-data` CSS media query detects whether the user has a preference for using less network traffic. For example, you can use this media query to avoid loading large font files and use a system font instead.
```